temp/AKSeImageTemplateWindows.json

```diff
@@ -0,0 +1,187 @@
+        },
+        "github_branch_name": {
+            "type": "string",
+            "defaultValue": "master"
+        },
+        "location": {
+            "type": "string"
+        },
+        "vm_size": {
+            "type": "string"
+        },
+        "build_number": {
+            "type": "string"
+        },
+        "build_id": {
+            "type": "string"
+        },
+        "build_repo": {
+            "type": "string"
+        },
+        "git_version": {
+            "type": "string"
+        }
+    },
+    "resources": [
+        {
+            "name": "[parameters('template_name')]",
+            "type": "Microsoft.VirtualMachineImages/imageTemplates",
+            "apiVersion": "2022-02-14",
+            "location": "[parameters('location')]",
+            "dependsOn": [],
+            "tags": {
+                "userIdentity": "enabled"
+            },
+            "identity": {
+                "type": "UserAssigned",
+                "userAssignedIdentities": {
+                    "[concat('/subscriptions/',parameters('subscription_id'),'/resourcegroups/',parameters('vhd_resource_group_name'),'/providers/Microsoft.ManagedIdentity/userAssignedIdentities/',parameters('managed_identity_name'))]": {}
+                }
+            },
+            "properties": {
+                "buildTimeoutInMinutes" : 180,
+                "vmProfile": 
+                    {
+                        "vmSize": "[parameters('vm_size')]",
+                        "osDiskSizeGB": 127,
+                        "vnetConfig": {
+                            "subnetId": "[concat('/subscriptions/',parameters('subscription_id'),'/resourcegroups/',parameters('vnet_resource_group_name'),'/providers/Microsoft.Network/virtualNetworks/',parameters('vnet_name'),'/subnets/',parameters('subnet_name'))]"
+                        }
+                    },
+                "source": {
+                    "type": "PlatformImage",
+                    "publisher": "MicrosoftWindowsServer",
+                    "offer": "WindowsServer",
+                    "sku": "2019-Datacenter-Core-smalldisk",
+                    "version": "17763.5329.231230"
+                },
+                "customize": [
+                    {
+                        "type": "PowerShell",
+                        "name": "configure-windows-vhd-phase1",
+                        "runElevated": true,
+                        "scriptUri": "[uri('https://raw.githubusercontent.com/Azure/aks-engine-azurestack/',concat(parameters('github_branch_name'),'/vhd/packer/configure-windows-vhd-phase1.ps1'))]"
+                    },
+                    {
+                        "type": "WindowsRestart",
+                        "restartCheckCommand": "echo Azure-Image-Builder-Restarted-the-VM  > c:\\buildArtifacts\\azureImageBuilderRestart.txt",
+                        "restartTimeout": "10m"
+                    },
+                    {
+                        "type": "WindowsRestart",

```

Comment:
do we need 2 restarts?

Reply:
Yes, need them to ensure configurations apply successfully, we have this in current flow as well (https://github.com/Azure/aks-engine-azurestack/blob/master/vhd/packer/windows-vhd-builder.json#L64C9-L71C11).